presto-docs/src/main/sphinx/release/release-342.md

```diff
@@ -0,0 +1,52 @@
+# Release 342 (XX Sep 2020)
+
+## General Changes
+
+* Improve performance of queries that use decimal type. ({issue}`4886`)
+* Improve of queries involving `IN` with subqueries by extending support for dynamic
+  filtering and dynamic partition pruning. ({issue}`5017`)
+* Improve dynamic partition pruning and query performance by reducing latency of dynamic filters collection. ({issue}`4988`)
+* Improve join performance when cost-based optimizer has missing or inaccurate stats. ({issue}`5141`)
```

Comment:
"inaccurate" or "incomplete"?

also, it's a table who has missing stats, not the cost-based optimizer, so:

"Improve performance of queries joining tables with missing or incomplete column statistics when the cost-based optimizer is enabled (which is the default)."

(doesn't sound great....)

Reply:
> also, it's a table who has missing stats, not the cost-based optimizer, so:

technically, intermediate cbo-stats stats can be inaccurate too. I'm fine with your version too